decide_designable.py

```diff
@@ -74,14 +74,22 @@ def nussinov(sequence,level:int):
     return False
 
 def generate_sequence(g: Graph, v: int) -> list[str]:
+    pairs = [("A", "U"), ("U", "A"), ("C", "G"), ("G", "C")]
+    letters = ['A', 'U', 'C', 'G']
 
+    outputs = []
     if g.neighborhood_size(v, 1, "out") == 1:
-        return ["AU", "UA", "CG", "GC"]
+        # children dots
+        for letter in letters:
+            output = letter * g.vs[v]["unpaired_count_0"]
+            for pair in pairs:
+                outputs.append(pair[0] + output + pair[1])
+
+        outputs = add_dots(g, v, outputs)
+        return outputs
 
```

Comment:
sprawdzajmy tutaj też tylko te sensowne literki
```suggestion
    pairs = [("A", "U"), ("U", "A"), ("C", "G"), ("G", "C")]
    letters = ['C', 'G', 'A', 'U']

    outputs = []
    if g.neighborhood_size(v, 1, "out") == 1:
        # children dots
        for i in {0, 2}:
            for letter in letters[i:i+2]:
                output = letter * g.vs[v]["unpaired_count_0"]
                for pair in pairs[i:i+2]:
                    outputs.append(pair[0] + output + pair[1])

        outputs = add_dots(g, v, outputs)
        return outputs
```

Reply:
dodałem tylko  

for i in {0, 2}

oraz

[i:i+2] (dwa razy)

oraz

zmieniłem kolejność w tablicy letters